gymnasium/dev_wrappers/clip_action.py

```diff
@@ -0,0 +1,41 @@
+"""Wrapper for clipping actions within a valid bound."""
+import jumpy as jp
+
+import gymnasium as gym
+from gymnasium import spaces
+from gymnasium.core import ActType
+
+
+class ClipActionV0(gym.ActionWrapper):
+    """Clip the continuous action within the valid :class:`Box` observation space bound.
+
+    Example:
+        >>> import gymnasium as gym
+        >>> import numpy as np
+        >>> env = gym.make('BipedalWalker-v3', disable_env_checker=True)
+        >>> env = ClipActionV0(env)
+        >>> env.action_space
+        Box(-1.0, 1.0, (4,), float32)
+        >>> env.step(np.array([5.0, 2.0, -10.0, 0.0]))
+        # Executes the action np.array([1.0, 1.0, -1.0, 0]) in the base environment
+    """
+
+    def __init__(self, env: gym.Env):
+        """A wrapper for clipping continuous actions within the valid bound.
+
+        Args:
+            env: The environment to apply the wrapper
+        """
+        assert isinstance(env.action_space, spaces.Box)
+        super().__init__(env)
```

Comment:
Technically the action space can be expanded to `Box(-np.inf, npinf, env.action_space.shape)` as the action space is the valid inputs rather than output.

Reply:
I see :+1: so no problem in changing the action space output with respect to the old `clip_action` wrapper?

Reply:
I don't think so